pkg/scheduler/plugin.go

```diff
@@ -188,17 +188,18 @@ func (b *binder) prepareTaskForLaunch(ctx api.Context, machine string, task *pod
 	pod.Annotations[annotation.BindingHostKey] = machine
 	task.SaveRecoveryInfo(pod.Annotations)
 
-	for _, entry := range task.Spec.PortMap {
-		oemPorts := pod.Spec.Containers[entry.ContainerIdx].Ports
-		ports := append([]api.ContainerPort{}, oemPorts...)
-		p := &ports[entry.PortIdx]
-		p.HostPort = int(entry.OfferPort)
-		op := strconv.FormatUint(entry.OfferPort, 10)
-		pod.Annotations[fmt.Sprintf(annotation.PortMappingKeyFormat, p.Protocol, p.ContainerPort)] = op
-		if p.Name != "" {
-			pod.Annotations[fmt.Sprintf(annotation.PortNameMappingKeyFormat, p.Protocol, p.Name)] = op
+	for _, m := range task.Spec.PortMap {
```

Comment:
No, but the thinking was that we should be able to roll back if needed. If you overwrite the original values then that's not possible.